Automattic/newspack-theme
# Add option to hide page title

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds an option to hide the page title on individual pages on the front-end.

Closes #1048.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Edit a page; confirm that you have the option to hide the title under Status & Visibility, and that it's off by default.

![image](https://user-images.githubusercontent.com/177561/104078905-50c67780-51d5-11eb-82a6-4ea1025b1a0b.png)

3. Enable 'Hide page title', and click 'Update'.
4. Confirm that the page title is not displaying: 

![image](https://user-images.githubusercontent.com/177561/104079028-d64a2780-51d5-11eb-91a7-9d4afe4e875c.png)

5. Toggle the Hide page title option back on, and confirm that the title displays again. 

### Other information:
